### PR TITLE
Add support for ACCEL_LOCATION udev property to deal with 2 sensors

### LINKED ISSUE #6557: Add "orientation ignore" property for accelerometers
From Hans:
> -Talked to Bastien about what to do with accelerometers in the base of a
>     clamshell design (mini)laptop the conclusion was that iio-sensor-proxy
>     should create an input-dev for it for games, but otherwise it should be
>     ignored (auto rotation makes no sense on a non-yoga clamshell, esp. if
>     the sensor is not even in the display part).

We should probably add some information about the physical location of the accelerometer in the device, but it might be worth checking whether the ACPI _PLD already contains this information (so we can script this).

CC @jwrdegoede @dsd

### PR DESCRIPTION
**Pull request by ljmf00**
This pull requests fixes #6557 .

As referred to in [this](https://lore.kernel.org/patchwork/patch/1057409/) kernel patch, `ACCEL_LOCATION` property should be added to udev.

---
Related issues and pull requests:
- [#166](https://github.com/hadess/iio-sensor-proxy/issues/166) on iio-sensor-proxy.
- [#262](https://github.com/hadess/iio-sensor-proxy/pull/262) on iio-sensor-proxy.

### CODE REVIEW

File: hwdb/60-sensor.hwdb [hunk 1]
```diff
@@ -57,6 +57,11 @@
 # automatically flip their output for an upside-down display when the device
 # is held upright.
 #
+#    ACCEL_LOCATION=<location>
+#
+# where <location> is the location of the sensor. This value could be 'base'
+# or 'display'. The default, when unset, is equivalent to:
```
Comment (jwrdegoede) [inline hwdb/60-sensor.hwdb:63]: It looks like you forgot to add some text here after "The default, when unset, is equivalent to:" I guess that should read: "The default, when unset, is equivalent to: 'display'"

Comment (ljmf00) [reply hwdb/60-sensor.hwdb:63]: Thanks. It's fixed now

**Comment by poettering** [comment]
(btw, we don't use "Signed-of-by" in systemd. though this doesn't matter)

**Comment by poettering** [comment]
Hmm, if this is useful, shouldn't this PR also contain a hwdb addition for some actual hw that makes use of the new prop?

**Comment by poettering** [comment]
(patch looks ok to me btw. but would prefer if @jwrdegoede and/or @hadess would have a look first, and merge it)

**Comment by jwrdegoede** [comment]
@poettering the changes look good to me, these changes go hand in hand with: https://github.com/hadess/iio-sensor-proxy/pull/262 once both are in place (and thus I can test) I've at least one model device which needs this, it has an accelerometer in the base, which should not be be used by iio-sensor-proxy to rotate the display.

Also there are several yoga-style devices with accelerometers in both the display and the base(keyboard) parts and iio-sensor-proxy needs to know which is which. So I expect actual users of this new prop to show up soon enough.

I don't have the rights to merge this, but from my pov this PR is ready to be merged.

**Comment by poettering** [comment]
@jwrdegoede you have the privs now, if you accept the github invitation. Use your new powers wisely! (usually we require one review from someone else to merge something. And of course the CI to pass, except when it fails, and is unrelated)

**Pull request merged**